cmd/juju/application/bundle.go

```diff
@@ -374,6 +376,19 @@ func (h *bundleHandler) getChanges() error {
 		return errors.Trace(err)
 	}
 
+	// Filter create offer changes if the feature flag is not enabled. We
+	// need to do it here rather in handleChanges() as the bundle handler
+	// will also iterate the list to print out a changelog.
+	if !featureflag.Enabled(feature.CMRAwareBundles) {
+		var filtered []bundlechanges.Change
+		for _, ch := range changes {
+			if ch.Method() != "createOffer" {
```

Comment:
I should probably have used type sniffing here. However, this bit of code is temporary and will be removed together with the feature flag once all bits land.